pandas/core/arrays/numpy_.py

```diff
@@ -131,14 +133,14 @@ def _from_sequence(
             result = result.copy()
         return cls(result)
 
-    def _from_backing_data(self, arr: np.ndarray) -> PandasArray:
+    def _from_backing_data(self, arr: np.ndarray) -> NumpyExtensionArray:
         return type(self)(arr)
 
     # ------------------------------------------------------------------------
     # Data
 
     @property
-    def dtype(self) -> PandasDtype:
+    def dtype(self) -> NumpyEADtype:
```

Comment:
Would be good to note that the returned `dtype` has changed too

Reply:
sure.  thoughts on the name NumpyEADtype?  i was torn between wanting parity with ArrowDtype vs confusion with np.dtype

Reply:
IMO NumpyEADtype sounds good